.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint Code
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ env.node_version }}
+        uses: actions/setup-node@v2
+        with:
```

Comment:
We can remove the `with: ` setting as we’re not using a matrix

Reply:
ah, no we still need to tell it which version of Node to use, but you are right that this env value was missing

Reply:
It defaults to LTS, but if you wanted to specify you could just add:

```yaml
  with:
    node-version: '12.x'
```

Reply:
Or what you did above! 🙂